PowerShell/ScubaGear/Modules/Orchestrator.psm1

```diff
@@ -147,6 +163,20 @@ function Invoke-SCuBA {
             return
         }
 
+        if ($PSCmdlet.ParameterSetName -eq 'Import-Configuration'){
+            Get-ScubaConfig -Path $ConfigFilePath
+            $ProductNames = $ScubaConfig.ProductNames
+            $M365Environment = $ScubaConfig.M365Environment
+            $OPAPath = $ScubaConfig.OPAPath
+            $LogIn = $ScubaConfig.LogIn
+            $DisconnectOnExit = $ScubaConfig.DisconnectOnExit
+            $OutPath = $ScubaConfig.OutPath
+            $OutFolderName = $ScubaConfig.OutFolderName
+            $OutProviderFileName = $ScubaConfig.OutProviderFileName
+            $OutRegoFileName = $ScubaConfig.OutRegoFileName
+            $OutReportName = $ScubaConfig.OutReportName
```

Comment:
1) Does ScubaConfig really need to be at global scope?  It seems to me that it could be at local scope and explicit passed as an argument into functions where needed

2) What if we inverted this and in the case of the argument-based parameter set populated scube config based on arguments.  We could then pass individual parameters to downstream functions, but incrementally move to passing the scubaconfig object as a single parameter object to clean up the code in cases where a large number of parameters were used.

Reply:
1) I changed scope to local - that is what I had in comments as my initial intent
2) Inverted.  Made changs needed in Invoke-SCuBA only.  We should open another issue or advice team to convert use single config object, $ScubaConfig, when touching applicable code.  I believe we will not even have to pass and just use it like a service and directly access at lower scopes.